kotlintest-assertions/src/main/kotlin/io/kotlintest/inspectors/funcs.kt

```diff
@@ -16,22 +16,29 @@ fun <T> runTests(col: Collection<T>, f: (T) -> Unit): List<ElementResult<T>> {
 }
 
 fun <T> buildAssertionError(msg: String, results: List<ElementResult<T>>): String {
-
   val passed = results.filterIsInstance<ElementPass<T>>()
   val failed = results.filterIsInstance<ElementFail<T>>()
+  val maxOutputResult = System.getProperty("kotlintest.assertions.maxOutputResult")?.toInt() ?: 10
```

Comment:
Shouldn't this new variable be documented?

Reply:
I think I should. I write document as method's comment.
ec5caf3